CMakeLists.txt

```diff
@@ -174,7 +180,7 @@ if(NO_XWAYLAND)
     add_compile_definitions(NO_XWAYLAND)
 else()
     message(STATUS "XWAYLAND Enabled (NO_XWAYLAND not defined) checking deps...")
-    pkg_check_modules(xdeps REQUIRED IMPORTED_TARGET xcb xwayland xcb-util xcb-render xcb-image xcb-xfixes xcb-icccm xcb-composite xcb-res xcb-ewmh)
+    pkg_check_modules(xdeps REQUIRED IMPORTED_TARGET xcb xwayland xcb-util xcb-render xcb-xfixes xcb-icccm xcb-composite xcb-res xcb-ewmh)
```

Comment:
why is xcb-image removed?

Reply:
As explained in the commit message, it hasn't been used by wlroots for ~6 years.

Reply:
but wlroots wouldn't compile for me without it.

Reply:
Can you quote the error message? CI is green here.

Reply:
hm, odd. When I was making the original switch there were some messages referring to missing xcb-image symbols, but you are right, removing it works.